frontend/templates/troubleshooting/index.tsx

```diff
@@ -114,21 +114,26 @@ const Wiki: NextPageWithLayout<{
                   borderBottom="1px"
                   borderColor="gray.300"
                >
-                  <Hide below="sm">
-                     <Image
-                        src={mainImageUrl}
-                        onClick={onOpen}
-                        cursor="pointer"
-                        alt={title}
-                        htmlWidth={120}
-                        htmlHeight={90}
-                        objectFit="contain"
-                        borderRadius="md"
-                        outline="1px solid"
-                        outlineColor="gray.300"
-                        marginRight={3}
-                     />
-                  </Hide>
+                  <Image
+                     sx={{
+                        display: 'none',
+                        // sm breakpoint
+                        '@media (min-width: 576px)': {
+                           display: 'block',
+                        },
```

Comment:
Could we do:

```
const smBreakpoint = useToken('breakpoints', 'sm');

...

sx={{
                        display: 'none',
                        // sm breakpoint
                        `@media (min-width: smBreakpoint)`: {
                           display: 'block',
                        },
```

This would reduce the use of the token being a one-off magic number

Reply:
We could do this:
```
   const foo: any = {
      display: 'none',
   };
   foo[`@media (min-width: ${smBreakpoint})`] = {
      display: 'block',
   };
```

Reply:
I think I lean that direction. It's less readable, which stinks, but it is slightly cleaner code IMO.

Reply:
[a81cd4f](https://github.com/iFixit/react-commerce/pull/1860/commits/a81cd4f359ae3588913099e44189694d04abfe6d)